extensions/amp-inline-gallery/1.0/README.md

```diff
@@ -0,0 +1,319 @@
+
+#### Layout and style
+
+Each Bento component has a small CSS library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
+
+```html
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-accordion-1.0.css">
+```
+
+Alternatively, you may also make the light-weight pre-upgrade styles available inline:
+
+```html
+<style data-bento-boilerplate>
+  amp-inline-gallery,
+  amp-inline-gallery-pagination,
+  amp-inline-gallery-thumbnails {
+    display: block;
+  }
+  amp-inline-gallery {
+    contain: layout;
+  }
+  amp-inline-gallery-pagination,
+  amp-inline-gallery-thumbnails {
+    overflow: hidden;
+    position: relative;
+  }
+  amp-inline-gallery-pagination:not(.i-amphtml-built) > :not([placeholder]):not(.i-amphtml-svc),
+  amp-inline-gallery-thumbnails:not(.i-amphtml-built) > :not([placeholder]):not(.i-amphtml-svc) {
+    display: none;
+    content-visibility: hidden;
+  }
+</style>
+```
+
+#### Attributes
+
+##### `<bento-inline-gallery-pagination>`
+
+###### `inset`
+
+Default: `false`
+
+Boolean attribute indicating whether to display the pagination indicator as inset (overlaying the carousel itself)
+
+##### `<bento-inline-gallery-thumbnails>`
+
+###### `aspectRatio`
+
+Optional
+
+Number: ratio of width to height that slides should be displayed in.
+
+###### `loop`
+
+Default: `false`
+
+Boolean attribute indicating whether thumbnails should loop.
+
+#### Styling
+
+You may use the `bento-inline-gallery`, `bento-inline-gallery-pagination`, `bento-inline-gallery-thumbnails`, and `bento-base-carousel` element selectors to style the pagination indicator, thumbnails, and carousel freely.
+
+### Preact/React Component
+
+The examples below demonstrates use of the `<BentoInlineGallery>` as a functional component usable with the Preact or React libraries.
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-inline-gallery
+```
+
+```javascript
+import React from 'react';
+import { BentoInlineGallery } from '@ampproject/bento-inline-gallery/react';
+import '@ampproject/bento-inline-gallery/styles.css';
+
+function App() {
+  return (
+    <BentoInlineGallery id="inline-gallery">
+      <BentoInlineGalleryThumbnails aspect-ration="1.5" loop></BentoInlineGalleryThumbnails>
+      <BentoBaseCarousel snap-align="center" visible-count="1.2" loop>
+        <img src="server.com/static/inline-examples/images/image1.jpg">
+        <img src="server.com/static/inline-examples/images/image2.jpg">
+        <img src="server.com/static/inline-examples/images/image3.jpg">
+      </BentoBaseCarousel>
+      <BentoInlineGallery-pagination inset></BentoInlineGallery-pagination>
+    </BentoInlineGallery>
+  }
+```
+
+[/example]
+
+#### Interactivity and API usage
```

Comment:
same here, this can be removed, including `actions` and `events`

Reply:
removed